crates/bevy_ecs/src/schedule/executor/multi_threaded.rs

```diff
@@ -112,6 +112,12 @@ pub struct MultiThreadedExecutor {
     panic_payload: Arc<Mutex<Option<Box<dyn Any + Send>>>>,
     /// When set, stops the executor from running any more systems.
     stop_spawning: bool,
+    /// Cached tracing spans for system tasks
+    #[cfg(feature = "trace")]
+    system_task_spans: Vec<Span>,
```

Comment:
Should this be a field on SystemTaskMetadata? Or does this need mutable access to use properly?

Reply:
I'll try moving it to there.